book/07-git-tools/sections/interactive-staging.asc

```diff
@@ -21,14 +21,14 @@ $ git add -i
 What now>
 ----
 
-You can see that this command shows you a much different view of your staging area than you're probably used to -- basically, the same information you get with `git status` but a bit more succinct and informative.
-It lists the changes you’ve staged on the left and unstaged changes on the right.
+Makikita mo na ang utos na ito ay nagpapakita sayo ng isang mas magkaiba na tingnan sa iyong staging na lugar kayasa sa marahil na ginagamit sa -- talagang, ang parehong impormasyon na iyong makukuha gamit ang `git status` na ngunit medyo maikli at nakakapagturo.
+Ito ay naglilista ng mga pagbabago na iyong nai-stage sa kaliwa at mga naka-unstage na mga pagbabago sa iyong kanan.
```

Comment:
"mas magkaiba na tingnan sa iyong staging na lugar kayasa sa marahil na ginagamit sa" I think its better to say it this way "mas kakaibang hitsura ng iyong staging na lugar kaysa nakagawiang paggamit nito"

Reply:
"gamit ang `git status` na ngunit medyo maikli at nakakapagturo" I suggest to change it to "gamit ang `git status` kung saan ay medyo maikli at makabuluhan"

Reply:
"nai-stage sa kaliwa at mga naka-unstage" I suggest to change it in this way "na-stage sa kaliwa at mga hindi na stage"

Reply:
done change.. followed your suggestions.